Add method to detect couriers for tracking number

bertvandormael opened the pull request.

ludeeus commented:
Can you add a sample response or documentation that has that response for the added endpoint in this PR?

bertvandormael commented:
@ludeeus I added an example for the new method to the example.py file. I also added the possibility to pass optional parameters to the method.
You can find some more info on this endpoint on their API docs: https://docs.aftership.com/api/4/couriers/post-couriers-detect.

ludeeus commented:
I will merge this now, but a new release will take some time.
